lib/app/constant/error_code.dart

```diff
@@ -5,6 +5,8 @@ class ErrorCode {
   /// For Example
   static const USER_NOT_EXIST = "E1";
     static const USER_SUSPEND = "E2";
+  static const IMAGE_EXCEEDS_SIZE_LIMIT = "E3";
```

Comment:
is this already aligned with the backend?

Reply:
@Farrize please fix this 🦏